docs/workflow-messaging.md

```diff
@@ -0,0 +1,77 @@
+## Workflow queues and topics - questions/decisions
+
+There are two SNS topics owned by storage:
+
+- born-digital-bag-notifications-prod
+- born-digital-bag-notifications-staging
+
+There will be two new SNS topics for Goobi:
+
+- digitised-bag-notifications-prod
+- digitised-bag-notifications-staging
+
+_(I assume, or similar names anyway)_
+
+There are/will be queues subscribed to these topics. Where do the queues live? We have currently one production queue, known as:
+
+arn:aws:sqs:eu-west-1:975596993436:born-digital-notifications-prod (in storage acct)
+arn:aws:sqs:eu-west-1:653428163053:born-digital-notifications-prod (in digirati acct)
+
+There is also similar for staging. 
+
+The WorkflowProcessor listens to just this queue at the moment, because we've only tested the one environment and it's only for born-digital events.
+
+Will we have another pair of queues for prod and staging that recieve messages from the digitised-bag-notifications-xxx topic? So that a running WorkflowProcessor is picking up from two queues?
+Or should we have one queue that subscribes to both born-digital and digitised topics, and gets workflow messages from both? (should rename that queue if that's the case).
```

Comment:
I think 2 queues as it's a lot simpler. 1 single queue wouldn't really work for pausing from one source.

We could use messageFiltering to pause from one origin but you'd lose delivery of the message. If we use the "open and look" then the paused-origin messages would eventually be sent to DLQ as the maxReceives would be hit fairly quick (and non-paused origin messages would increasingly be slow to process as there'd be lots of ones we're not interested in).